src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,164 @@
+  <string name="no_matching_applications_found">Keine passenden Apps gefunden</string>
+  <string name="no_proxy">Kein Proxy</string>
+  <string name="notify_about_updates">Update-Benachrichtigung</string>
+  <string name="notify_about_updates_summary">Bei verfügbaren Updates eine Benachrichtigung anzeigen</string>
+  <string name="number_of_applications">Anzahl der Apps</string>
+  <string name="ok">OK</string>
+  <string name="only_compatible_with_FORMAT">Nur kompatibel mit %s</string>
+  <string name="only_on_wifi">Nur über WLAN</string>
+  <string name="open_DESC_FORMAT">Öffne %s?</string>
+  <string name="other">Andere</string>
+  <string name="parsing_index_error_DESC">Indexdatei konnte nicht analysiert werden.</string>
+  <string name="password">Passwort</string>
+  <string name="password_missing">Passwort fehlt</string>
+  <string name="permissions">Berechtigungen</string>
+  <string name="plus_more_FORMAT">+%d mehr</string>
+  <string name="preferences">Einstellungen</string>
+  <string name="processing_FORMAT">Verarbeite %1$s</string>
+  <string name="project_website">Projekt-Webseite</string>
+  <string name="promotes_non_free_network_services">Bewirbt unfreie Netzwerkservices</string>
+  <string name="promotes_non_free_software">Bewirbt unfreie Software</string>
+  <string name="provided_by_FORMAT">Bereitgestellt von %s</string>
+  <string name="proxy">Proxy</string>
+  <string name="proxy_host">Proxy-Host</string>
+  <string name="proxy_port">Proxy-Port</string>
+  <string name="proxy_type">Proxy-Typ</string>
+  <string name="repositories">Paketquellen</string>
+  <string name="repository">Paketquelle</string>
+  <string name="repository_not_used_DESC">Diese Paketquelle wurde noch nicht verwendet. Bitte Aktivieren um die angebotenen Apps anzusehen.</string>
+  <string name="repository_unsigned_DESC">Unsigniert. App-Liste konnte nicht verifiziert werden. Vorsicht bei Downloads aus unsignierten Paketquellen.</string>
+  <string name="requires_FORMAT">Benötigt %s</string>
+  <string name="save">Speichern</string>
+  <string name="saving_details">Speichere Details</string>
+  <string name="screenshots">Screenshots</string>
+  <string name="search">Suche</string>
+  <string name="select_mirror">Wähle einen Mirror</string>
+  <string name="show_more">Zeige mehr</string>
+  <string name="show_older_versions">Zeige ältere Versionen</string>
+  <string name="signature_FORMAT">Signatur %s</string>
+  <string name="signed_using_unsafe_algorithm">Signiert mit unsicherem Algorithmus</string>
+  <string name="skip">Überspringen</string>
+  <string name="socks_proxy">Socks-Protokoll</string>
+  <string name="sorting_order">Sortierreihenfolge</string>
+  <string name="source_code">Quellcode</string>
+  <string name="source_code_no_longer_available">Quellcode nicht mehr verfügbar</string>
+  <string name="suggested">Empfohlen</string>
+  <string name="sync_repositories">Synchronisiere Paketquellen</string>
```

Comment:
```suggestion
  <string name="sync_repositories">Synchronisier Paketquellen</string>
```
Why is the ending "e" allowed in imperative form? https://de.wiktionary.org/wiki/synchronisieren

Reply:
It is indeed allowed and more common with an e at the end. Would rather leave it as is. 😀